docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+    merged_single_cells = convert(
+        source_path=(
+            "s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace"
+            "/backend/2020_11_04_CPJUMP1/BR00117012/BR00117012.sqlite"
+        ),
+        dest_path="./BR00117012.parquet",
+        dest_datatype="parquet",
+        source_datatype="sqlite",
+        chunk_size=8000,
+        preset="cellprofiler_sqlite_cpg0016_jump",
+        no_sign_request=True,
+        # use explicit cache to avoid temp cache removal
+        local_cache_dir="./sqlite_s3_cache/",
+        parsl_config=Config(
+            executors=[ThreadPoolExecutor(label="tpe_for_jump_processing")]
+        ),
+        sort_output=False,
+    )
+else:
+    merged_single_cells = "./BR00117012.parquet"
+
+# read only the metadata from parquet file
+parquet.ParquetFile(merged_single_cells).metadata
+
+# + editable=true slideshow={"slide_type": ""}
+# show the first few columns for metadata column names
+schema = parquet.read_schema(merged_single_cells)
+print("\n".join(str(schema).split("\n")[:12]))
+
+# +
+# set a list of metadata columns for use throughout
+metadata_cols = [
+    "Metadata_ImageNumber",
+    "Image_Metadata_Row",
+    "Image_Metadata_Site",
+    "Metadata_ObjectNumber",
+    "Metadata_Plate",
+    "Metadata_Well",
+    "Image_TableNumber",
+]
+
+# read only metadata columns with outlier-specific columns
+df_merged_single_cells = pd.read_parquet(
+    path=merged_single_cells,
+    columns=[
+        *metadata_cols,
+        "Nuclei_AreaShape_Area",
+        "Nuclei_AreaShape_FormFactor",
+        "Nuclei_AreaShape_Eccentricity",
+    ],
+)
+df_merged_single_cells.head()
+# -
+
+# label outliers within the dataset
+print("Large nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="large_nuclei",
+)
+
+# label outliers within the dataset
+print("Elongated nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="elongated_nuclei",
+)
+
+# label outliers within the dataset
+print("Small and low formfactor nuclei outliers:")
+df_labeled_outliers = cosmicqc.analyze.find_outliers(
+    df=df_merged_single_cells,
+    metadata_columns=metadata_cols,
+    feature_thresholds="small_and_low_formfactor_nuclei",
+)
+
+# label outliers within the dataset
+df_labeled_outliers = cosmicqc.analyze.label_outliers(
+    df=df_merged_single_cells,
+    include_threshold_scores=True,
+)
+# show added columns
+df_labeled_outliers[
+    [col for col in df_labeled_outliers.columns.tolist() if "cqc." in col]
+].head()
+
+# +
+# create a column which indicates at least one outlier being found
+df_labeled_outliers["analysis.included_at_least_one_outlier"] = df_labeled_outliers[
+    [col for col in df_labeled_outliers.columns.tolist() if ".is_outlier" in col]
+].any(axis=1)
```

Comment:
What does it mean for "at least one outlier being found"? Is this per condition? If the data is at a single-cell level, then wouldn't this just be "is_outlier"? I would recommend adding more background for this column.

Reply:
Thanks for checking on this - this creates a column which indicates whether "any" outlier was found from all outlier threshold sets within coSMicQC. I did this to help summarize the results in a single plot, instead of showing for example a single threshold set. I'll add in a few more notes to this effect.